app/javascript/packs/billboard.js

```diff
@@ -0,0 +1,25 @@
+import { setupDisplayAdDropdown } from '../utilities/displayAdDropdown';
+
+// the term billboard can be synonymously interchanged with displayAd
+async function getBillboard() {
+  const placeholderElement = document.getElementsByClassName(
+    'js-display-ad-comments-container',
+  )[0];
+
+  const { articleId, placementArea } = placeholderElement.dataset || {};
+
+  if (placeholderElement.innerHTML.trim() === '') {
+    const response = await window.fetch(
+      `/display_ads/for_display?article_id=${articleId}&placement_area=${placementArea}`,
```

Comment:
These params will need to be added to `config/fastly/snippets/safe_params_list.vcl` (this is the allow list for params which the edge cache does not ignore — I feel like we may want to get rid of this altogether, as it has proven to be a pain to keep up). If we don't add to that list, these extra query params will be disregarded by the edge cache :)

Reply:
✅ Thanks for this tip!

Reply:
🎉